examples/package.json

```diff
@@ -37,7 +37,7 @@
     "@tensorflow/tfjs-backend-cpu": "^4.4.0",
     "@upstash/redis": "^1.20.6",
     "@vercel/kv": "^0.2.3",
-    "@xata.io/client": "^0.25.1",
+    "@xata.io/client": "^0.28.0",
```

Comment:
this is not consistent with the version in the langchain subproject, fixes a type error that manifested